auto_round/compressors/base.py

```diff
@@ -2750,7 +2750,6 @@ def _quantize_blocks(
                 modules = [get_module(model, n) for n in names]
                 m = WrapperMultiblock(modules)
 
-            m = m.to(device)
```

Comment:
Where is the code that moves the block to GPU when device_map=0? I haven’t seen it.
Also, please run the default command (auto-round --model xxx)for the 8B model to check whether the speed and accuracy are typically same with the main branch.